### PR TITLE
Translated into German | Habe die Einstellungen ins Deutsche übersetzt :)

### PR DESCRIPTION
**Pull request by nsde**
Hope I can support this project this way :) you're awesome, I love this modpack

### CODE REVIEW

**Comment by nsde** [comment]
Let me know if I should shorten some lines!

File: Mod Menu Helper/assets/modmenu/lang/de_de.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+    "modmenu.dropInfo.line1": "FO mods haben §9✎§r oder §8✎§r, Alternativen haben §c*§r;",
+    "modmenu.dropInfo.line2": "andere werden nicht unterstützt",
+    "modmenu.summaryTranslation.slight-gui-modifications": "Animationen, Modpack-Version im Menü\n§9✎§8 Animationen, Farben, FPS-Zähler, ...",
+    "modmenu.summaryTranslation.advancementinfo": "Große und elegante Fortschritt-Übersicht\n§9✎§8 Farben, Info-Box Größe, ...",
+    "modmenu.summaryTranslation.animatica": "Animierte Texturen von Optifine\n§8✎ (nur fortgeschrittene Optionen)",
+    "modmenu.summaryTranslation.antighost": "Behebe verbuggte Blöcke mit /ghost\n§9✎§8 Taste",
+    "modmenu.summaryTranslation.bettermounthud": "Zeigt Hungerkeulen und XP an, während man auf einem Pferd sitzt\n§8✎ (keine Optionen)",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/de_de.json:8]: The first line is too long, cuts off after "auf"

File: Mod Menu Helper/assets/modmenu/lang/de_de.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "modmenu.summaryTranslation.capes": "Zeigt Optifine-Umhänge sowie andere an\n§9✎§8 Typen, bevorzugter Typ, Elytra, ...",
+    "modmenu.summaryTranslation.citresewn": "Einstellbare Gegenstände, wie bei Optifine\n§9✎§8 Mod ein-/ausstellen, kaputte Pakete erlauben, ...",
+    "modmenu.summaryTranslation.colormatic": "Einstellbare Farben, wie bei Optifine\n§9✎§8 Klarer Himmel und klare Leere, Lichtglättung",
+    "modmenu.summaryTranslation.continuity": "Verbundene Texturen, woe bei Optifine\n§9✎§8 Mod ein-/ausstellen, Methode, Ressourcenpakete",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/de_de.json:12]: Second line cuts off after "Methode"

File: Mod Menu Helper/assets/modmenu/lang/de_de.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    "modmenu.summaryTranslation.cullleaves": "\"Bessere Bäume\", wie bei Optifine\n§9✎§8 Mod ein-/ausstellen, Ressourcenpaket",
+    "modmenu.summaryTranslation.cem": "Einstellbare Enitäten, wie bei Optifine (Alpha)\n§9✎§8 Optifine wird unterstützt, behobenes Modell",
+    "modmenu.summaryTranslation.dcch": "Chatverlauf hinweg über Welten und Servern behalten\n§8✎ (keine Optionen)",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/de_de.json:15]: First line cuts off after "Servern"

File: Mod Menu Helper/assets/modmenu/lang/de_de.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    "modmenu.summaryTranslation.dynamicfps": "FPS im Hintergrund verringern\n§9✎§8 Mod ein-/ausstellen, bei Maushover zurücksetzen, ...",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/de_de.json:16]: Second line cuts off after "Maushover"

File: Mod Menu Helper/assets/modmenu/lang/de_de.json [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    "modmenu.summaryTranslation.enhancedblockentities": "Optimiert Truhen, Schilder, Betten usw.\n§9✎§8 zu optimierende Blöcke, Methode",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/de_de.json:17]: Does the second line have to start with a small letter?

File: Mod Menu Helper/assets/modmenu/lang/de_de.json [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    "modmenu.summaryTranslation.entityculling": "Verhindert das Rendern von Mobs hinter Mauern\n§8✎ (nur fortgeschrittene Optionen)",
+    "modmenu.summaryTranslation.fabrishot": "HD Screenshots mit F9\n§9✎§8 Überschreibe F2, Auflösung, Taste, ...",
+    "modmenu.summaryTranslation.fastchest": "§c*§r Optimierte Truhen\n§9✎§8 Mod ein-/ausstellen",
+    "modmenu.summaryTranslation.ferritecore": "Verringert Arbeitsspeicherauslastung\n§8✎ nur fortgeschrittene Optionen)",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/de_de.json:21]: First parenthesis missing on second line

File: Mod Menu Helper/assets/modmenu/lang/de_de.json [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    "modmenu.summaryTranslation.hydrogen": "Verringert (auch) Arbeitsspeicherauslastung\n§8✎ keine Optionen)",
+    "modmenu.summaryTranslation.iris": "Optifine-Shader Unterstützung\n§9✎§8 Mod ein-/ausstellen, Shader auswählen, Taste, ...",
+    "modmenu.summaryTranslation.itemmodelfix": "Fix transparent lines in held items\n§9✎§8 Method",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/de_de.json:24]: Untranslated

File: Mod Menu Helper/assets/modmenu/lang/de_de.json [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+    "modmenu.summaryTranslation.lazydfu": "Schnellerer Spielstart\n§8✎ (keine Optionen)",
+    "modmenu.summaryTranslation.lithium": "Optimiert Spielphysik und Chunks\n§8✎ (nur fortgeschrittene Optionen)",
+    "modmenu.summaryTranslation.modmenu": "Dieses Modmenü\n§9✎§8 Knopfeinstellungen, Filtern, Sortieren, ...",
+    "modmenu.summaryTranslation.morechathistory": "Erhöht das Charverlauf-Limit\n§8✎ (keine Optionen)",
+    "modmenu.summaryTranslation.nofade": "Entfernt Lade/Verblass-Animationen\n§8✎ (keine Optionen)",
+    "modmenu.summaryTranslation.no-telemetry": "Deaktiviert das Senden von Diagnostikdaten\n§8✎ (keine Optionen)",
+    "modmenu.summaryTranslation.notenoughcrashes": "Erlaubt das weitere Spielen nach einem Absturz\n§8✎ (nur fortgeschrittene Optionen)",
+    "modmenu.summaryTranslation.phosphor": "Optimiert Licht, schnelleres Laden von Chunks\n§8✎ (nur fortgeschrittene Optionen)",
+    "modmenu.summaryTranslation.smoothboot": "Verhindert Spielabstürze\n§8✎ (nur fortgeschrittene Optionen)",
+    "modmenu.summaryTranslation.sodium": "Optimiert das Rendern, mehr FPS\n§9✎§8 In den Videoeinstellungen: Optimierungen",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/de_de.json:37]: "Videoeinstellungen" should be "Grafikeinstellungen" as that is how it's translated in German

File: Mod Menu Helper/assets/modmenu/lang/de_de.json [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+    "modmenu.summaryTranslation.reeses-sodium-options": "Vertikale Videoeinstellungen\n§8✎ (keine Optionen)",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/de_de.json:38]: Same as above

File: Mod Menu Helper/assets/modmenu/lang/de_de.json [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+    "modmenu.summaryTranslation.sodium-extra": "Grafikoptionen von Optifine\n§9✎§8 In den Videoeinstellungen: Animationen, Partikel, ...",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/de_de.json:39]: Same as above

File: Mod Menu Helper/assets/modmenu/lang/de_de.json [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+    "modmenu.summaryTranslation.tooltipfix": "Verhindert Tooltip-Überläufe\n§8✎ (keine Optionen)",
+    "modmenu.summaryTranslation.wi_zoom": "Zoome, indem du die Taste C drückst\n§9✎§8 Benutze das Mausrad, um zu zoomen - Taste",
+    "modmenu.summaryTranslation.yosbr": "Standardeinstellungen des Modpacks\n§9✎§8 Schau ins FO-Wiki, falls du sie zurücksetzen möchtest."
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/de_de.json:42]: Second line cuts off at "zürücksetzen"

**Review by Madis0 — CHANGES_REQUESTED** [review]
Some changes are needed, you can test the language by yourself too:
https://github.com/Fabulously-Optimized/fabulously-optimized/wiki/Language-support#translating-mod-menu-helper

**Comment by nsde** [comment]
Alright, should be fine now.

**Comment by Madis0** [comment]
Thank you!

**Comment by nsde** [comment]
You're welcome. Is it done now? :)

**Comment by Madis0** [comment]
I merged it, so it will be in the next version I release, whenever that is.

**Pull request merged**